src/scrapers.js

```diff
@@ -22,23 +25,34 @@ exports.detailPageFunction = ($universities) => {
     const data = [];
 
     $universities.forEach(($university) => {
-        const uniStatsSelector = $university.querySelectorAll('.uni_stats > .container .key');
-        const location = $university.querySelector('.lead_section .location');
         const result = {};
-
-        result.title = $university.querySelector('.lead_section .title_info h1').innerText;
-        result.address = location && location.innerText ? location.innerText.replace(' View map', '').replace(/\n/g, ' ') : null;
-        result.mapLink = location ? location.querySelector('a').href : null;
-        result.logoLink = $university.querySelector('.lead_section .logo_area img').src;
-
-        // University stats
-        uniStatsSelector.forEach((item) => {
-            const label = item.querySelector('label').innerText.replace(/\s/g, '');
-            const formattedLabel = label.charAt(0).toLowerCase() + label.slice(1);
-            const value = item.querySelector('.val').innerText;
-            result[formattedLabel] = value;
+        const locations = $university.querySelectorAll('.desktop-view .campus-locations');
+        const locationsResult = [];
+        locations.forEach((locationNode) => {
+            locationsResult.push(locationNode.textContent.trim());
```

Comment:
this is having a "\n" inside, maybe split between address/country? there are two spans inside. the array can be like 

```js
locations.forEach((locationNode) => {
  const [location, country] = locationNode.querySelectorAll('span');
  locationsResult.push({
    country: country?.textContent?.trim(),
    mapLink: locationNode.href,
    location: location?.textContent?.trim().replace(/,$/, ''),
  });
});
```